provider/image.go

```diff
@@ -428,6 +436,37 @@ func getCredentials() (map[string]clitypes.AuthConfig, error) {
 	return auths, nil
 }
 
+func getRegistryAddr(serverName, imgName string) (string, error) {
+	var serverAddr string
+	if serverName == "docker.io" {
+		// if it's dockerhub, we special case it so host config can find the correct registry
+		return "https://index.docker.io/v1/", nil
+	}
```

Comment:
Other tools in the CNI space omit the schema and assume https://, such as [`oras`](https://github.com/oras-project/oras/), [`skopeo`](https://github.com/containers/skopeo), and even Kubernetes.